src/txadmission.cpp

```diff
@@ -442,7 +443,7 @@ void ThreadTxAdmission()
                             static const unsigned int nMaxOrphanTx = (unsigned int)std::max(
                                 (int64_t)0, GetArg("-maxorphantx", DEFAULT_MAX_ORPHAN_TRANSACTIONS));
                             static const uint64_t nMaxOrphanPoolSize = (uint64_t)std::max(
-                                (int64_t)0, (GetArg("-maxmempool", DEFAULT_MAX_MEMPOOL_SIZE) * 1000000 / 10));
+                                (int64_t)0, (GetArg("-maxmempool", DEFAULT_MAX_MEMPOOL_SIZE) * 1000000 / 2));
                             unsigned int nEvicted = orphanpool.LimitOrphanTxSize(nMaxOrphanTx, nMaxOrphanPoolSize);
```

Comment:
not sure i agree with making the orphanpool so big.  I feel like we could open up an out of memory attack vector here, or possibly cause nodes with less available memory to run out of memory.  If we want to change this then I think we should have a separate PR which gives us a new option -maxorphanpool, so the users can override and configure it if necessary.